plugin/src/main/java/org/opensearch/ml/autoredeploy/MLModelAutoReDeployer.java

```diff
@@ -0,0 +1,307 @@
+/*
+ *
+ *  * Copyright OpenSearch Contributors
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.opensearch.ml.autoredeploy;
+
+import static org.opensearch.ml.common.CommonValue.ML_MODEL_INDEX;
+import static org.opensearch.ml.settings.MLCommonsSettings.*;
```

Comment:
We usually don't import `*`. You can configure in the intellij so that it doesn't import all. 

https://blog.marcnuri.com/intellij-idea-how-to-disable-wildcard-imports